integration-libs/cds/src/schematics/add-cds/index_spec.ts

```diff
@@ -94,42 +99,72 @@ describe('Spartacus CDS schematics: ng-add', () => {
         .toPromise();
     });
 
-    it('should not create the feature module', () => {
-      const featureModule = appTree.readContent(spartacusCdsFeatureModulePath);
-      expect(featureModule).toBeFalsy();
+    it('should not create any of the feature modules', () => {
+      expect(appTree.exists(featureModulePath)).toBeFalsy();
     });
-    it('should not add the feature to the feature module', () => {
-      const spartacusFeaturesModule = appTree.readContent(
-        'src/app/spartacus/spartacus-features.module.ts'
-      );
-      expect(spartacusFeaturesModule).toMatchSnapshot();
+
+    it('should install necessary Spartacus libraries', () => {
+      const packageJsonContent = appTree.readContent('package.json');
+      const dependencies = JSON.parse(packageJsonContent).dependencies;
```

Comment:
I think we should merge here devDependencies with dependencies

Reply:
Done